src/conversion/model-to-view-converters.js

```diff
@@ -272,6 +271,127 @@ export function unwrap( elementCreator ) {
 }
 
 /**
+ * Function factory, creates a converter that converts new model marker to view attribute element.
+ *
+ * Let's assume following model and view. `{}` represents a range that is added as a marker with `searchResult` name.
+ * The range represents a result of search `ab` string in the model document. The range has to be visualized in view.
+ *
+ *		[paragraph]              MODEL ====> VIEW        <p>
+ *			|- {a                                         |- <span class="searchResult">
+ *			|-  b}                                        |   |- ab
+ *			|-  c                                         |- c
+ *
+ * The wrapping node depends on passed parameter. If {@link module:engine/view/attributeelement~AttributeElement} was passed, it
+ * will be cloned and the copy will become the wrapping element. If `Function` is provided, it is passed all the parameters of the
+ * {@link module:engine/conversion/modelconversiondispatcher~ModelConversionDispatcher#event:addMarker addMarker event}. It's expected
+ * that the function returns a {@link module:engine/view/attributeelement~AttributeElement}. The result of the function will be the
+ * wrapping element. When provided `Function` does not return element, then will be no conversion.
+ *
+ * The converter automatically consumes corresponding value from consumables list, stops the event (see
+ * {@link module:engine/conversion/modelconversiondispatcher~ModelConversionDispatcher}).
+ *
+ *		modelDispatcher.on( 'addMarker:searchResult', wrapRange( new ViewAttributeElement( 'span', { class: 'searchResult' } ) ) );
+ *
```

Comment:
I would like to see here the difference between `wrapItem` and this, so no one in the future asks again can we merge these two converters. I would be nice to have it written directly: "In contrast to `wrapItem`...".